data/the emerald sword.txt

```diff
@@ -0,0 +1,796 @@
+				`	"I'm sorry, but I think nothing good can come out of this. Better to forget about the cube."`
+					decline
+			`	"Knowing their methods, they'll likely find you first. But don't be alarmed, they're not... evil, per se. Just be careful, alright?"`
+				accept
+	
+	to complete
+		never
+
+mission "The Box 4"
+	landing
+	name "The box"
+	description "Confront the mercenaries who stole the cube on Watcher."
+	source "Stormhold"
+	destination "Watcher"
+	
+	to offer
+		has "The Box 3: offered"
+	
+	on offer
+		conversation
+			branch visited
+				has "visited mammon"
+			branch first
+				not "visited mammon"
+			label first
+			`You arrive safely on Stormhold, despite the pirates clearly not wanting you here. You're still halfway through the atmosphere, and someone hails you, with a robotic, modulated voice.`
+			`	"I heard you were looking for me, <first> <last>," the person on the other end says.`
+			choice
+				`	"Who are you? How do you know my name?"`
+					goto who
+				`	"You did? How?"`
+			`	"In my line of business, I have to know everything. Including all of my clients. How else would you hope to find me?"`	
+				goto intro
+			label who
+			`	The mysterious person laughs.`
+			`	"It looks like I know you more than you know yourself. I am who you're looking for, of course."`
+			label intro
+			`	"Mammon. Information dealer. At your service." The voice speaks ominously.`
+			`	"Come meet me in person to discuss the specifics of what you're looking for."`
+			`	The connection cuts, and you receive coordinates to somewhere apparently random, in the middle of the sea.`
+			`	You arrive at the proposed location in the middle of nowhere, under stormy conditions. Initially, you question whether you received the right coordinates or not, until a submarine emerges from the waves. It is a huge vessel, and lights guide you to land on it's deck, which your spaceship comfortably fits in. Armed men in business suits surround your ship, and when you unboard, two of them guide you inside. You're led into a dimly lit room with a holographic map of the galaxy on the ceiling and a desk, behind which a hooded figure sits.`
+			`	"Please, take a seat," they say. Their voice has a strange tone with no distinguishable gender, with a sybilating, guttural accent, and it's not until you take a seat that you notice a reptilian snout poking out of the hood. You don't know whether this is natural or an extreme body modification.`
+			choice
+				`(Say nothing.)`
+					goto middle
+				`"Are you a..."`
+			`	They cut you off. "Who or what I am doesn't matter to you."`
+			label middle
+			`	"Let's get to what matters. You want to find that alien artifact, yes?"`
+			`	You nod in response.`
+			`	"I have the information you need. Finding lost objects is the most trivial of my services, but as with everything, it has a price.`
+			`	"Ten million credits." they say. "Non-negotiable. And don't try anything funny."`
+			branch cash
+				credits >= 12000000
```

Comment:
I left this with more than the real amount because the payment is made elsewhere after the mission. This means the player could arrive with exactly 10M, accept the quest, and with mortgage or crew payments, go under 10M and avoid the payment quest. It would of course still be offered after the player has enough money again and take it all, but it could be months and would feel weird. I'm unable to put the payment in this quest since it needs to still be offered if the player doesn't have the money, and if i did that, it wouldn't get offered unless the player had more than 10M. So I worked around the code limitations.

Reply:
Given that the payment *is* in a separate mission, I'd say the dialog from Mammon needs to change up a bit. Right now you give them the credit chips, then they give you the information, making it seem as though they accepted them, but then you later receive the mission saying the credits were withdrawn, which seems very confusing. 

How about making it so that Mammon declines the credit chips that you try to give to them, instead saying "I will make my withdrawal when I desire. I only required that you accept paying," or something to that effect. Then, the payment mission could perhaps only trigger at a random point after this mission is over and the player has >12 million, but the player can accept to pay even if that have only >10 million. In that way a player with >10 million can accept, but they won't be made broke because of the >12 million trigger for the 10 mil to be withdrawn.

Reply:
Interesting idea. This also makes mammon seem to be even more influential than now. I'll try to do that.